cache.yaml

```diff
@@ -37,13 +37,10 @@ cache:
   # store-type: badgerdb
   store-type: badgerdb
   # directory to be used by the persistence store
-  dir: ./cached/caches
+  dir: /home/mava/projects/cachev2/cache
```

Comment:
Can you revert the dir back to ./cached/caches ? Or what It needs to be ?

Reply:
cheers, reverted it